backend/src/ml_space_lambda/data_access_objects/app_configuration.py

```diff
@@ -0,0 +1,382 @@
+#
+#   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from __future__ import annotations
+
+import json
+from typing import Dict, List, Optional
+from venv import logger
+
+from dynamodb_json import json_util as dynamodb_json
+
+from ml_space_lambda.data_access_objects.dynamo_data_store import DynamoDBObjectStore
+from ml_space_lambda.enums import ResourceType
+from ml_space_lambda.utils.mlspace_config import get_environment_variables
+
+
+class AppConfigurationModel:
```

Comment:
Would using a Type be more appropriate than using a Class? I would think that if using a Class we would have specific methods for this class but it seems we just store values here. It would also get rid of those to_dict methods I think. Same thing applies to all the other Classes in this file

I know you are probably modeling this based on the other DAO so I guess its open to discussion

Reply:
We need the to_dict and from_dict methods to swap between the data types to store this and then parse it from DDB. Are you seeing a way we can sidestep needing those methods?